extensions/wikia/AdEngine/js/video/vastUrlBuilder.js

```diff
@@ -55,7 +56,7 @@ define('ext.wikia.adEngine.video.vastUrlBuilder', [
 		options = options || {};
 		slotParams = slotParams || {};
 
-		var correlator = Math.round(Math.random() * 10000000000),
+		var correlator = options.correlator ? options.correlator : Math.round(Math.random() * 10000000000),
```

Comment:
We need common correlator for few vastUrls and in other case we want just to build single vastUrl without care about that